openqabot/loader/repohash.py

```diff
@@ -21,11 +22,14 @@ def get_max_revision(
 ) -> int:
     max_rev = 0
 
-    url_base = f"http://download.suse.de/ibs/{project.replace(':',':/')}"
+    url_base = f"{OBS_DOWNLOAD_URL}/{project.replace(':',':/')}"
 
     for repo in repos:
+        # handle URLs for SLFO specifically
+        if project == "SLFO":
+            url = f"{OBS_DOWNLOAD_URL}/{repo[0].replace(':',':/')}:/{repo[1].replace(':',':/')}/standard/repodata/repomd.xml"
         # openSUSE and SLE incidents have different handling of architecture
-        if repo[0].startswith("openSUSE"):
+        elif repo[0].startswith("openSUSE"):
```

Comment:
this should stay as `if repo[0] ....` , it has nothing to do with previous condition